packages/@react-aria/combobox/src/useComboBox.ts

```diff
@@ -103,7 +103,8 @@ export function useComboBox<T>(props: AriaComboBoxOptions<T>, state: ComboBoxSta
     shouldFocusWrap,
     ref: inputRef,
     // Prevent item scroll behavior from being applied here, should be handled in the user's Popover + ListBox component
-    isVirtualized: true
+    isVirtualized: true,
+    shouldUseVirtualFocus: true
```

Comment:
wanna keep this change still even though I'm not doing the useSelectableCollection change anymore , feels like something we missed before